tests/Console/Command/StageCommandTest.php

```diff
@@ -30,22 +47,88 @@ public function testBasicConfiguration(): void
         $definition = $command->getDefinition();
         $arguments = $definition->getArguments();
         $options = $definition->getOptions();
+        $composerCommandArgument = $definition->getArgument('composer-command');
 
         self::assertSame('stage', $command->getName(), 'Set correct name.');
         self::assertSame([], $command->getAliases(), 'Set correct aliases.');
         self::assertNotEmpty($command->getDescription(), 'Set a description.');
-        self::assertSame([], array_keys($arguments), 'Set correct arguments.');
+        self::assertSame(['composer-command'], array_keys($arguments), 'Set correct arguments.');
         self::assertSame([], array_keys($options), 'Set correct options.');
+        self::assertNotEmpty($command->getUsages(), 'Set usages.');
+        self::assertNotEmpty($command->getHelp(), 'Set help.');
+
+        self::assertTrue($composerCommandArgument->isRequired(), 'Required Composer command option.');
+        self::assertTrue($composerCommandArgument->isArray(), 'Required Composer command option.');
+        self::assertNotEmpty($composerCommandArgument->getDescription(), "Description provided.");
     }
 
     /**
      * @covers ::execute
+     *
+     * @dataProvider providerBasicExecution
      */
-    public function testBasicExecution(): void
+    public function testBasicExecution($composerCommand, $stagingDir, $output): void
```

Comment:
Should these arguments be type hinted?

Reply:
I don't have a strong opinion; I like to use return type hints everywhere for consistency, but this is totally a stylistic thing that isn't important.